Components/navigation/navigation.css

```diff
@@ -0,0 +1,47 @@
+.cui-navigation-container{
+    width: 100%;
+    height: 5rem;
+    display: flex;
+    align-items: center;
+    padding: 0 2rem;
+}
+
+.cui-nav-brand{
+    display: flex;
+    align-items: center;
```

Comment:
you can put display: flex code in a single class and use it anywhere else

Reply:
Yes, I will surely do that when I add utility classes during refactoring.
For now, I'm keeping it like this.

Reply:
Okay then you are good to go

Reply:
okay , looks good to merge